atom/app/node_main.cc

```diff
@@ -5,6 +5,7 @@
 #include "atom/app/node_main.h"
 
 #include <memory>
+#include <string>
```

Comment:
Perhaps if strings only needed on MAC OS, might want to put the include in the ifdef.

Reply:
Not really necessary